tools/taskcluster/upload_symbols.sh

```diff
@@ -0,0 +1,9 @@
+#/bin/sh
+export PYTHONPATH="tools/taskcluster/library/python/redo:tools/taskcluster/library/python/requests"
+export OBJCOPY="tools/taskcluster/dist/arm-linux-androideabi-objcopy"
+chmod +x "tools/taskcluster/dist/dump_syms"
+python ./tools/taskcluster/crashreporter/symbolstore.py -c --vcs-info -s ./tools/taskcluster ./tools/taskcluster/dist/dump_syms ./tools/taskcluster/crashreporter/crashreporter-symbols ./app/build/intermediates/cmake/googlevr/release/obj/armeabi-v7a/libnative-lib.so
+python ./tools/taskcluster/crashreporter/symbolstore.py -c --vcs-info -s ./tools/taskcluster ./tools/taskcluster/dist/dump_syms ./tools/taskcluster/crashreporter/crashreporter-symbols ./app/build/intermediates/cmake/noapi/release/obj/armeabi-v7a/libnative-lib.so
+python ./tools/taskcluster/crashreporter/symbolstore.py -c --vcs-info -s ./tools/taskcluster ./tools/taskcluster/dist/dump_syms ./tools/taskcluster/crashreporter/crashreporter-symbols ./app/build/intermediates/cmake/wavevr/release/obj/armeabi-v7a/libnative-lib.so
+python ./tools/taskcluster/crashreporter/symbolstore.py -c --vcs-info -s ./tools/taskcluster ./tools/taskcluster/dist/dump_syms ./tools/taskcluster/crashreporter/crashreporter-symbols ./app/build/intermediates/cmake/svr/release/obj/armeabi-v7a/libnative-lib.so
```

Comment:
Please help confirm if the path `./app/build/intermediates/cmake/{DEVICE_NAME}/release/obj/` is correct in Taskcluster. I can't reproduce them in my local.

Reply:
You can use https://tools.taskcluster.net/tasks/create to test your changes.